### PR TITLE
Update scorecard to ubuntu font

### PR DESCRIPTION
**Pull request by kicha0**

### CODE REVIEW

File: src/responsibleai/rai_analyse/_score_card/common_components.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_full_html(htmlbody):
 def get_css():
     return """<style>
         * {
-          font-family: Segoe UI;
+          font-family: Ubuntu;
```
Comment (vinuthakaranth) [inline src/responsibleai/rai_analyse/_score_card/common_components.py:35]: nit: can you please add in description why this change is needed? like if PM requested it or are we keeping in compliance with something? thanks!

**Review by vinuthakaranth — APPROVED** [review]

**Comment by kicha0** [comment]
Designer requested change of font for scorecard rendering.

File: src/responsibleai/rai_analyse/_score_card/common_components.py (hunk 1 above)
Comment (kicha0) [reply src/responsibleai/rai_analyse/_score_card/common_components.py:35]: Designer requested change of font for scorecard rendering.
